tools/runtime-stats-monitor.md

```diff
@@ -0,0 +1,215 @@
+
+To use the default Panel Settings:
+
+1. From the **Project** tab, go to **Packages** > **Multiplayer Tools** > **NetStatsMonitor** > **Implementation** > **UI** > **Resources**. The default panel settings configuration for the RNSM is in this folder: `UnityMpToolsRnsmDefaultPanelSettings.asset`.
+2. Select the `GameObject` with the RNSM component so that it appears in the **Inspector** tab.
+3. Drag the default panel setting asset from the **Project** tab to the **Panel Settings Override** field under the **RuntimeNetStatsMonitor** component of your `GameObject`.
+
+To configure your own Panel Settings presets:
+
+1. Select the **Assets** folder under the **Project** tab.
+2. Right-click in the **Assets** folder, then navigate to **Create** > **UI Toolkit** and click on **Panel Settings Asset**.
+3. The configuration script appears under the **Assets** folder and also opens in the **Inspector** tab.
+4. From the **Inspector** tab, you can adjust each of the fields to fit your screen needs.
+5. Once configured, you can add your Panel Settings preset to your RNSM `GameObject` by selecting the the RNSM `GameObject` to appear in the **Inspector** tab.
+6. Your **Assets** folder should still be visible in the **Project** tab, so select your newly created Panel Settings to drag and drop over the **Panel Settings Override** field under the **RuntimeNetStatsMonitor** component in the **Inspector** tab.
+7. **Save** your scene.
+
+### Styling the Runtime Network Stats Monitor
+
+You can customize the styling of the RNSM by dragging and dropping a Unity Style Sheet (USS) onto the **Custom Style Sheet** field of the RNSM component from the **Inspector** tab.
+
+For more information about Unity Style Sheets (USS), see the Unity Documentation on [Styling the UI with USS](https://docs.unity3d.com/2021.2/Documentation/Manual/UIE-USS.html).
+
+You can use the default USS file for the RNSM to style your RNSM or use it as a starting point for custom styling. It can be found from the **Project** tab under **Packages** > **Multiplayer Tools** > **NetStatsMonitor** > **Implementation** > **UI** > **Resources** > **UnityMpToolsRnsmDefaultStyleSheet.uss**. Then drag it over the **Custom Style Sheet** field in the **Inspector** tab.
+
+To create a new style sheet for the RNSM:
+
+1. Select the **Assets** folder under the **Project** tab.
+2. Right-click in the **Assets** folder, then navigate to **Create** > **UI Toolkit** and click on **Style Sheet**.
+3. The new USS file appears under the **Assets** folder and also opens in the **Inspector** tab.
+4. From the **Inspector** tab, you can use the + or - symbols to adjust each style element you want to include in this preset. Each field is configurable from here.
```

Comment:
It might be worth noting that because it's so similar to CSS which some users are likely to be familiar with, it may be more intuitive for some users to create or modify this file as a text file rather than using the inspector.